auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -336,7 +336,7 @@ export default class EntityManagement extends Mixins(AccountChangeMixin, NextPag
 
   // create a numbered company
   protected async startNumberedCompany () {
-    await this.createNumberedBusiness(this.currentAccountSettings.id)
+    await this.createNumberedBusiness(this.currentOrganization.id)
```

Comment:
currentOrganization.id works for both staff and user to return a valid id unlike currentAccountSettings